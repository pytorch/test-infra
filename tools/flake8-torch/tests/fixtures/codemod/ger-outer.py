import torch
deprecated = torch.norm()
sinusoid_inp = torch.ger(pos_seq, inv_freq)
other = something.ger(pos_seq, inv_freq)
deprecated = torch.norm()
