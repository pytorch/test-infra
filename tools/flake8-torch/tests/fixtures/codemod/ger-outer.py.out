import torch
deprecated = torch.norm()
sinusoid_inp = torch.outer(pos_seq, inv_freq)
other = something.ger(pos_seq, inv_freq)
deprecated = torch.norm()
